scripts/actions/__tests__/check-for-outdated-translations.test.js

```diff
@@ -0,0 +1,78 @@
+const fs = require('fs');
+const fetch = require('node-fetch');
+
+const checkForOutdatedTranslations = require('../check-for-outdated-translations');
+
+// Mock node-fetch so we can avoid calling out to GitHub
+jest.mock('node-fetch');
+
+global.process.exit = jest.fn();
+
+// Helper function to mock a response from Github
+const mockGithubResponse = (result) => {
+  fetch.mockResolvedValueOnce({
+    json: jest.fn(() => Promise.resolve(result)),
```

Comment:
adding an `ok` field here should fix the unit tests

Reply:
whoops! thanks :D